modules/sbt-dependency-manager/build.sbt

```diff
@@ -36,6 +36,11 @@ libraryDependencies <++= scalaBinaryVersion {
   case "2.11" => ("com.ning" % "async-http-client" % "[1.6.5, 1.6.5]" force() exclude("org.jboss.netty", "netty"))::Nil
 }
 
+libraryDependencies ++= Seq(
+  "io.get-coursier" %% "coursier" % "1.0.0-RC3",
```

Comment:
there's only 4 small tests that use `specs2`, the other use `scalatest`. 🗑'ing it,  seems reasonable :)